flaskphiid/annotation.py

```diff
@@ -161,6 +185,26 @@ def add_annotation(self, ann):
             self.type_map = self.type_map or ann.type_map
         self.source_annotations.append(ann)
 
+    def split_annotations_by_subtypes(self):
+        if len(self.source_types) == 1:
+            return self.source_annotations
```

Comment:
it seems like this is the only place we're not explicitly saying that there is a list of annotations. will it return a list?